string-analysis_test.go

```diff
@@ -139,6 +139,20 @@ func TestStringsSimilarity(t *testing.T) {
 		{"Cosine : Sentence 2", args{"I love horror movies", "Lights out is a horror movie", Cosine}, 0.20412414, false},
 		{"Cosine : Sentence 3", args{"love horror movies", "Lights out horror movie", Cosine}, 0.28867513, false},
 
+		// Jaccard method
+		{"Jaccard : First arg empty", args{"", "abcde", Jaccard}, 0.0, false},
+		{"Jaccard : Second arg empty", args{"abcde", "", Jaccard}, 0.0, false},
+		{"Jaccard : Same args", args{"abcde", "abcde", Jaccard}, 1.0, false},
+		{"Jaccard : No characters match", args{"abcd", "effgghh", Cosine}, 0.0, false},
```

Comment:
Idk if this case is useful for Jaccard but anyways it use Cosine algorithm instead of Jaccard one